packages/cli/src/cli.ts

```diff
@@ -1,6 +1,6 @@
 import { Option, program } from "@commander-js/extra-typings"
 import type { add, generate, remove, update } from "./commands"
-import { WellKnownChain } from "./well-known-chains"
+import * as knownChains from "@polkadot-api/known-chains"
```

Comment:
I was as well thinking why we have just a few known chains here, now it makes more sense 👍🏻